database/eventlog/00_create_schema.sql

```diff
@@ -12,8 +12,8 @@ CREATE TABLE zmon_eventlog.events(
   e_data jsonb
 );
 
-CREATE INDEX ON zmon_eventlog.events USING gin (e_data);
-CREATE INDEX ON zmon_eventlog.events (e_created , e_type_id);
+CREATE INDEX ON zmon_eventlog.events (((e_data->'checkId')::text), e_created);
+CREATE INDEX ON zmon_eventlog.events (((e_data->'alertId')::text), e_created);
 
```

Comment:
imho it does not matter that much, we query basically for all event types with alertId or checkId